forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
no, that is what it used to do and that is why labels on certain elements disappeared on page change.

when a page change happens the view is emptied but the reference to $label is still kept on the backbone view. Therefore on first render we need to check if $label exists, if not, create the label and store a reference to it. I did try and put this into the initialize function but it broke so many tests i left it in the renderLabel funciton

next, on a page change, for certain elements, that label is still attached to the backbone view but not at the DOM. therefore you need to check if its attached to the DOM, and if its not, re-attach it. This was the reason some elements were not rendering the label.

finally render the text of the label.

I believe that there is a more serious issue, im not exactly sure what it is yet, but as this is a highly visible problem a quick fix is warranted.